packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -41,11 +41,27 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
         })();
       });
     }
-
     openUserProfile(opts.userProfileProps);
     return opts.actionCompleteCallback?.();
   };
 
+  const handleUserProfileActionClicked = (__experimental_startPath?: string) => {
+    if (opts.userProfileMode === 'navigation') {
+      return navigate(opts.userProfileUrl || '').finally(() => {
+        void (async () => {
+          await sleep(300);
+          opts.actionCompleteCallback?.();
+        })();
+      });
+    }
+    openUserProfile({
+      ...opts.userProfileProps,
+      ...(__experimental_startPath && { __experimental_startPath }),
```

Comment:
❓Shouldn't we update the types of UserProfileModalProps and  OrgProfileModalProps to include `__experimental_startPath` ?